docs/deployment/configuration/auth_setup.rst

```diff
@@ -564,6 +564,8 @@ Follow the steps in this section to configure `flyteadmin` to use an external au
 
                # 2. Optional: Set external auth server baseUrl if different from OpenId baseUrl.
                externalAuthServer:
+               # Replace this with your deployment URL.  It will be used by flyteadmin to validate the token audience
+                 allowedAudience: [https://<your-flyte-deployment-URL>]
```

Comment:
since this is an array, in order to decrease the risk of confusion with the square brackets (e.g. should I include them or not?), can you put this in a separate line?

Reply:
do you mean something like this?:
```
allowedAudience: [
    https://<your-flyte-deployment-URL>
]
```

Reply:
@eapolinario or maybe just removing the brackets altogether?